src/main/java/org/generationcp/middleware/manager/ontology/OntologyVariableDataManagerImpl.java

```diff
@@ -1031,6 +1032,15 @@ public VariableOverrides getVariableOverridesByVariableIdAndProgram(final Intege
 		}
 	}
 
+	@Override
+	public List<VariableOverridesDto> getVariableOverridesByAliasAndProgram(final String alias, final String programUuid) {
+		try {
```

Comment:
try/catch in DAO should be enough @cuenyad

Reply:
done.